src/app/components/privacy-policy/privacy-policy.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit, Inject, HostListener } from '@angular/core';
-import { DOCUMENT } from '@angular/common';
+import { DOCUMENT } from '@angular/platform-browser';
```

Comment:
Any reason for this change ?

Reply:
Yes @Shekharrajak . Both the imports provide reference to the window object. But in this case `@angular/common` was not working as it was not picking the document body object. So, i have changed it to `@angular/platform-browser`.